gatsby/plugins/gatsby-source-cks/src/node-creation/topics.ts

```diff
@@ -36,7 +36,7 @@ export const createTopicNodes = (
 			const nodeContent = {
 				...topic,
 				...{
-					slug: slugify(topicName, { lower: true, remove: /[()']/g }),
+					slug: slugify(topicName, { lower: true, remove: /[(),']/g }),
```

Comment:
Great! Can you also write a test for both topics and specialities, (or extend the current ones) see https://github.com/nice-digital/cks-gatsby/blob/master/gatsby/plugins/gatsby-source-cks/src/node-creation/__tests__/topics.test.ts#L51